example/spec/parallel/dbRecords/DbRecordsDLQSpec.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const { fakeFileFactory, fakeGranuleFactoryV2 } = require('@cumulus/api/lib/testUtils');
+const { SQS } = require('@cumulus/aws-client');
+const { sqs } = require('@cumulus/aws-client/services');
+const { randomString } = require('@cumulus/common/test-utils');
+const { deleteS3Object, waitForObjectToExist } = require('@cumulus/aws-client/S3');
+
+const { loadConfig } = require('../../helpers/testUtils');
+
+describe('When a bad record is sent on the DLQ', () => {
```

Comment:
Originally I was testing sending a bad record on the input queue, but because SQS takes a really long time to wind through retries before sending things to the DLQ, and even updating the Queue's attributes takes at least a minute to propagate, I was not seeing test success within 5 minutes of waiting, so this component test sends the message directly to the DLQ. This does fall short of testing the full integration and whether the format of a message delivered to a DLQ is what we expect, but in ad-hoc testing I believe that it is (for example, on the input the sender is `aws:states` but on the DLQ it became `aws:sqs` because the sender now is the input queue.)

Reply:
Given the limitations here, that makes sense to me